src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/WorkforceAllocationsToDeliusApiClient.kt

```diff
@@ -152,9 +152,14 @@ data class DeliusCaseDetail(
 data class Event(val number: String)
 
 data class ProbationStatus(val description: String)
-data class InitialAppointment(val date: LocalDate?)
+data class InitialAppointment(val date: LocalDate?, val staff: Staff?)
+
+data class Staff @JsonCreator constructor(
+  val name: Name?,
+)
 data class DeliusCaseDetails(val cases: List<DeliusCaseDetail>)
-data class Name(val forename: String, val middleName: String?, val surname: String) {
+
+data class Name(val forename: String?, val middleName: String?, val surname: String?) {
```

Comment:
same again with `forename` and `surname` unless you have seen examples with these being null. Perhaps we should just ask the question on these when we confirm their API is working fine. As they need us to confirm that with them anyway :)